src/renderer/components/settings/Information.tsx

```diff
@@ -5,25 +5,29 @@
 // that can be found in the LICENSE file exposed on Github (readium) in the project repository.
 // ==LICENSE-END==
 
+import * as commonmark from "commonmark";
 import * as React from "react";
-
 import { connect } from "react-redux";
-
-import LibraryLayout from "readium-desktop/renderer/components/layout/LibraryLayout";
-
-import Header from "./Header";
-
 import { setLocale } from "readium-desktop/common/redux/actions/i18n";
-
-import { TranslatorProps, withTranslator } from "readium-desktop/renderer/components/utils/translator";
-
-import * as commonmark from "commonmark";
-
 import * as packageJson from "readium-desktop/package.json";
+import LibraryLayout from "readium-desktop/renderer/components/layout/LibraryLayout";
+import {
+    TranslatorProps, withTranslator,
+} from "readium-desktop/renderer/components/utils/translator";
+import * as info_de from "readium-desktop/resources/information/de.md";
+import * as info_en from "readium-desktop/resources/information/en.md";
+import * as info_fr from "readium-desktop/resources/information/fr.md";
+import Header from "./Header";
 
-import { readFile } from "fs";
+type Record<K extends keyof any, T> = {
+    [P in K]: T;
+};
 
-import { promisify } from "util";
+const info: Record<string, typeof import("*.md")> = {
```

Comment:
I think that defining a generic type just for the `lang` -> `info` mapping (which is effectively just a plain Javascript "indexed" object with string keys) introduces unnecessary complexity in the TypeScript source code.
Case in point, this line of code further down:
`info[locale] as unknown as string`

What about a simple type (name just for illustration purposes):
`interface StringToStringMap { [key: string]: string; }`

Reply:
you're right, done